doc/setup_ide.md

```diff
@@ -133,6 +134,34 @@ After the above steps:
 - Change run/debug `Working directory` for Linux configuration. (`Edit configurations…` -> `PurePhone`)
 - Change `Working directory` from `…/PurePhone/build` to `…PurePhone/build-linux`
 
+### Setting up in QtCreator
+
+- Install `QtCreator` version `3.5` or later
+- Install `CMake` version `3.19.5` or later
+- It is highly recommened that to install `Ninja` as it will decrease rebuild times.
```

Comment:
```suggestion
- It is highly recommened to install `Ninja` as it will decrease rebuild times.
```

Reply:
reccomended* :D

Reply:
Thank you for the corrections, I have incorporated them.  I'm just glad we no longer live in the age of typewriters.